core/capabilities/webapi/target/types.go

```diff
@@ -0,0 +1,30 @@
+package target
+
+import "github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/common"
+
+const (
+	AllAtOnce  string = "AllAtOnce"
+	SingleNode string = "SingleNode"
+)
+
+type Input struct {
+	URL     string            `json:"url"`               // URL to query, only http and https protocols are supported.
+	Method  string            `json:"method,omitempty"`  // HTTP verb, defaults to GET.
+	Headers map[string]string `json:"headers,omitempty"` // HTTP headers, defaults to empty.
+	Body    string            `json:"body,omitempty"`    // Base64-encoded binary body, defaults to empty.
+}
+
+// WorkflowConfig is the configuration of the workflow that is passed in the workflow execute request
+type WorkflowConfig struct {
+	TimeoutMs  uint32 `json:"timeoutMs,omitempty"`  // Timeout in milliseconds
+	RetryCount uint8  `json:"retryCount,omitempty"` // Number of retries, defaults to 0.
+	Schedule   string `json:"schedule,omitempty"`   // schedule, defaults to empty.
```

Comment:
Description doesn't say what this is.
Also, nit: not sure if schedule is the most accurate name for this. More like a strategy/type/method.

Reply:
how about `DeliveryMode` ?

Reply:
Yeah that sounds good